components/announcement_bar/notify_admin_downgrade_delinquency_bar/index.tsx

```diff
@@ -0,0 +1,138 @@
+    AnnouncementBarTypes, MappingCloudSelfHotsSkus, PaidFeatures, Preferences, TELEMETRY_CATEGORIES,
+} from 'utils/constants';
+
+import {GlobalState} from 'types/store';
+import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import useGetSubscription from 'components/common/hooks/useGetSubscription';
+import {isSystemAdmin} from 'mattermost-redux/utils/user_utils';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {trackEvent} from 'actions/telemetry_actions';
+import {getSubscriptionProduct} from 'mattermost-redux/selectors/entities/cloud';
+import {NotifyStatus, useGetNotifyAdmin} from 'components/common/hooks/useGetNotifyAdmin';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+
+import AnnouncementBar from '../default_announcement_bar';
+
+export const BannerPreferenceName = 'notify_upgrade_workspace_banner';
+
+const NotifyAdminDowngradeDelinquencyBar = () => {
+    const dispatch = useDispatch();
+    const getCategory = makeGetCategory();
+    const preferences = useSelector((state: GlobalState) =>
+        getCategory(state, Preferences.NOTIFY_ADMIN_UPGRADE_DOWNGRADE_WORKSPACE),
+    );
+    const subscription = useGetSubscription();
+    const product = useSelector(getSubscriptionProduct);
+    const currentUser = useSelector((state: GlobalState) =>
+        getCurrentUser(state),
+    );
+
+    const {notifyAdmin, notifyStatus} = useGetNotifyAdmin({});
+
+    useEffect(() => {
+        if (notifyStatus === NotifyStatus.Success) {
+            dispatch(savePreferences(currentUser.id, [{
+                category: Preferences.NOTIFY_ADMIN_UPGRADE_DOWNGRADE_WORKSPACE,
+                name: BannerPreferenceName,
+                user_id: currentUser.id,
+                value: 'adminNotified',
+            }]));
+        }
+    }, [currentUser, dispatch, notifyStatus]);
+
+    const shouldShowBanner = () => {
+        if (!subscription) {
+            return false;
+        }
+
+        if (isSystemAdmin(currentUser.roles)) {
+            return false;
+        }
+
+        if (!subscription.delinquent_since) {
+            return false;
+        }
+
+        if (!preferences) {
+            return false;
+        }
+
+        if (preferences.some((pref) => pref.name === BannerPreferenceName)) {
+            return false;
+        }
+
+        const delinquencyDate = new Date(subscription.delinquent_since * 1000);
+
+        const oneDay = 24 * 60 * 60 * 1000; // hours*minutes*seconds*milliseconds
+        const today = new Date();
+        const diffDays = Math.round(
+            Math.abs((today.valueOf() - delinquencyDate.valueOf()) / oneDay),
+        );
+
+        return diffDays > 90;
+    };
+
+    if (!shouldShowBanner() || product == null) {
+        return null;
+    }
+
+    const notifyAdminRequestData = {
+        required_feature: PaidFeatures.UPGRADE_DOWNGRADE_WORKSPACE,
+        required_plan: MappingCloudSelfHotsSkus[product?.sku || ''],
```

Comment:
Instead of this, wondering if you should
```suggestion
        required_plan: MappingCloudSelfHotsSkus[product?.sku] || '',
```

And then you can remove the `'':''` mapping from [here](https://github.com/mattermost/mattermost-webapp/pull/11119/files#diff-d01376ffc6b70925fed81daa8f355304ae72969f3923a6227017572c1b24f6f2R484)